RockLib.HealthChecks.AspNetCore/CHANGELOG.md

```diff
@@ -5,29 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.0-alpha.2 - 2023-02-28
+
+### Changed
+- Updated RockLib.HealthChecks package reference to `2.0.1`
+
 ## 3.0.0-alpha.1 - Not Yet Released
```

Comment:
You could probably put the date on here since it was already released.

Reply:
Done

Reply:
I don't see that this is updated.  For clarity I was talking about '3.0.0-alpha.1 - Not Yet Released'.

Reply:
Oof stupid Visual Studio showed an error message when I pushed it that I just noticed. But I'll push it once we get the build issues taken care of